config/jest.config.js

```diff
@@ -0,0 +1,39 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
+  clearMocks: true,
+  collectCoverageFrom: ['<rootDir>/src/**/*.{js,jsx}', '!<rootDir>/src/index.jsx'],
+  coverageDirectory: '<rootDir>/coverage',
+  coverageReporters: [
+    'text',
+    'lcov',
+    'clover',
+    'text-summary',
+  ],
+  coverageThreshold: {
+    global: {
+      // branches: 80,
+      // functions: 80,
+      // lines: 80,
+      // statements: 80,
+      branches: 0,
+      functions: 70,
+      lines: 60,
+      statements: 60,
+    },
+  },
+  rootDir: '../',
+  snapshotSerializers: ['enzyme-to-json/serializer'],
+  moduleDirectories: ['<rootDir>/node_modules'],
+  setupFilesAfterEnv: ['<rootDir>/config/jest.setup.js'],
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Non-HTML variable 'setupFilesAfterEnv' is used to store raw HTML](https://app.codacy.com/manual/iagoregiani/verificador-associacao/pullRequest?prid=5497764)